server/sonar-db-migration/src/main/java/org/sonar/server/platform/db/migration/def/TinyIntColumnDef.java

```diff
@@ -40,6 +40,8 @@ private TinyIntColumnDef(Builder builder) {
     super(builder.columnName, builder.isNullable, null);
   }
 
+  public static Builder newTinyIntColumnDefBuilder() { return new Builder(); }
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this line start at column 5. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)